source/_components/camera.bomradarloop.markdown

```diff
@@ -0,0 +1,167 @@
+---
+layout: page
+title: "BOM Radar Loop"
+description: "Instructions for integrating bomradarloop within Home Assistant"
+date: 2019-02-18 00:00
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Camera
+logo: home-assistant.png
```

Comment:
Please use a logo of the component and not of Home Assistant

Reply:
```suggestion
logo: bom.png
```

Reply:
Updated to use `bom.png`.